usr/share/rear/skel/default/etc/scripts/system-setup.d/58-start-dhclient.sh

```diff
@@ -44,23 +44,15 @@ for DEVICE in `get_device_by_hwaddr` ; do
 		ISALIAS=no
 	fi
 
-	# IPv4 DHCP clients
-	case $DHCLIENT_BIN in
-		(dhclient)
-			dhclient -lf /var/lib/dhclient/dhclient.leases.${DEVICE} -pf /var/run/dhclient.${DEVICE}.pid -cf /etc/dhclient.conf ${DEVICE}
-		;;
-		(dhcpcd)
-			dhcpcd ${DEVICE}
-		;;
-	esac
-
-	# IPv6 DHCP clients
-	case $DHCLIENT6_BIN in
-		(dhclient6)
-			dhclient6 -lf /var/lib/dhclient/dhclient.leases.${DEVICE} -pf /var/run/dhclient.${DEVICE}.pid -cf /etc/dhclient.conf ${DEVICE}
-		;;
-		(dhcp6c)
-			dhcp6c  ${DEVICE}
-		;;
+	case "$DHCLIENT_BIN" in
+		(dhclient*)
+			"$DHCLIENT_BIN" -lf /var/lib/dhclient/dhclient.leases.${DEVICE} -pf /var/run/dhclient.${DEVICE}.pid -cf /etc/dhclient.conf ${DEVICE}
+		    ;;
+		(dhcpcd*)
+			"$DHCLIENT_BIN" ${DEVICE}
+		    ;;
+		(*)
```

Comment:
@OliverO2 
I think I could just bindly and happily approve all your changes
because I see here that you added a default/fallback `case`
with an explanatory and helpful user info message
because it proves you do work carefully and in the end
that is all one can do and all what others can expect.